src/main/java/roomescape/Controller/PageController.java

```diff
@@ -0,0 +1,19 @@
+package roomescape.Controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class PageController {
+  @GetMapping("/")
+  public String home() {
+    return "home";
+  }
+  @GetMapping("/time")
+  public String time() { return "time"; }
```

Comment:
가독성을 위해 개행을 해보는건 어떨까요?
인텔리제이를 사용중이라면 `Ctrl + Alt + L` 로 라인 포맷팅을 할 수 있습니다

```suggestion
public class PageController {

  @GetMapping("/")
  public String home() {
    return "home";
  }

  @GetMapping("/time")
  public String time() {
    return "time"; 
  }
```

Reply:
써놓고 저장해놨는데 뺏겼어요 😢

Reply:
와 기가 막히네요!
f60ee27eadf42b41baf52b929574e6cea485faeb 적용했습니다